File: 프로그래머스/lv0/120869. 외계어 사전/외계어 사전.java
class Solution {
   public int solution(String[] spell, String[] dic) {

        for (int i = 0; i < dic.length; i++) {
            int answer = 0;
            for (int j = 0; j < spell.length; j++) {
                if (!dic[i].contains(spell[j])) {
                    break;
                } else {
                    answer++;
                    if (answer == spell.length) {
                        return 1;
                    }
                }
            }
            if (i == dic.length -1) {
                return 2;
            }
        }
        return 2;
    }
}

File: 프로그래머스/lv0/120869. 외계어 사전/외계어 사전.cs
using System;

public class Solution {
     public int solution(string[] spell, string[] dic)
        {
            int answer = 2;
            for (int i = 0; i < dic.Length; i++)
            {
                for (int j = 0; j < spell.Length; j++) // Fixed the loop index from i to j
                {
                    if (!(dic[i].Contains(spell[j])))
                    {
                        break;
                    }
                    dic[i] = dic[i].Replace(spell[j], "");

                    if (dic[i].Equals("") && j == spell.Length - 1)
                    {
                        answer = 1; // Set answer to 1 if the condition is met
                        break; // We can break here since we already know the answer is 1
                    }
                }
            }
            return answer;
        }

}